src/openrct2/scripting/Duktape.hpp

```diff
@@ -269,6 +275,11 @@ namespace OpenRCT2::Scripting
         return DukValue::take_from_stack(ctx);
     }
 
+    template<> inline DukValue ToDuk(duk_context* ctx, const std::string& value)
```

Comment:
I think its better to have std::string_view as the parameter here otherwise it might construct temporary std::strings when passing const char*

Reply:
I can't remove this as it causes compile errors, there is already another overload for string_view.

Reply:
I see, then it should be fine already.